readthedocs/projects/models.py

```diff
@@ -985,7 +985,7 @@ class Domain(models.Model):
     https = models.BooleanField(
         _('Use HTTPS'),
         default=False,
-        help_text=_('SSL is enabled for this domain')
+        help_text=_('Always use HTTPS for this domain')
```

Comment:
Technically this will result in a migration. However, there's a bunch of other minor pending migrations on `Project`.